A bug with loading the page by a slug

xartuu opened the pull request:
## Hello,
I described the bug already in June in [this discussion](https://discuss.flarum.org/d/18301-friendsofflarum-pages/229) on Flarum, but it has not been fixed, so I take it into my own hands.

#### I send a link to the [video](https://youtu.be/TEInlSW7zPc) where I present him.
I will describe here quickly what is on the film.
Before showing the bug I refreshed the page.
On the first tab of the browser I show what are the pages (slug, name, etc), on the second tab I show the first bug, and on the third tab I show the second bug.

#### I found a simple way to fix this bug. In pull request I send a commit containing a fix of this bug.

Commit:
Fixed a bug with loading the page by a slug

clarkwinkelmann commented:
Hi! Sorry for the long response time and thanks for the PR!

Could you update it to use single quotes, like we do across our codebase?

I discovered that `isNaN('') === false`, so this means that if a slug starts with `-`, the id will be `''` and it will be used as the value :thinking: So would decimal values like `12.4`. Maybe `isNaN(parseInt(id))` would be more reliable?

Commit:
Fixed a bug with loading the page by a slug

xartuu commented:
I changed the code and used single quotes, as you asked.

I did not pay attention to an error with finding the page with slug starting at `-`. I fixed it.

I sent a higher commit with the same name as the previous one.

clarkwinkelmann commented:
Thank you very much! We will probably merge this while preparing the beta 14 version and it'll be included in the beta14-compatible release.
